CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
```

Comment:
who is this global owner :godmode:

Reply:
left it in there intentionally for documentation...